### ISSUE TITLE
The build breaks with undefined `NGX_HTTP_V2_ENCODE_HUFF` and `NGX_HTTP_V2_ENCODE_RAW`.

### ISSUE THREAD

**Issue by StefanJum** [unikraft/lib-nginx#11]
When building the application with [Musl](), it gives this error:
```make
gbuild/libnginx/origin/nginx-1.15.6/src/http/v2/ngx_http_v2_encode.c:23:12: warning: implicit declaration of function ‘ngx_http_v2_huff_encode’; did you mean ‘ngx_http_v2_string_encode’? [-Wimplicit-function-declaration]                                                                                                                                          
   23 |     hlen = ngx_http_v2_huff_encode(src, len, tmp, lower);                                                                                                                                                  
      |            ^~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                         
      |            ngx_http_v2_string_encode                                                                                                                                                                       
gbuild/libnginx/origin/nginx-1.15.6/src/http/v2/ngx_http_v2_encode.c:26:16: error: ‘NGX_HTTP_V2_ENCODE_HUFF’ undeclared (first use in this function)                                                                                                                                                                                                                  
   26 |         *dst = NGX_HTTP_V2_ENCODE_HUFF;                                                                                                                                                                    
      |                ^~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                     
gbuild/libnginx/origin/nginx-1.15.6/src/http/v2/ngx_http_v2_encode.c:26:16: note: each undeclared identifier is reported only once for each function it appears in                                                                                                                                                                                                    
gbuild/libnginx/origin/nginx-1.15.6/src/http/v2/ngx_http_v2_encode.c:27:42: warning: implicit declaration of function ‘ngx_http_v2_prefix’; did you mean ‘ngx_http_v2_write_int’? [-Wimplicit-function-declaration]                                                                                                                                                   
   27 |         dst = ngx_http_v2_write_int(dst, ngx_http_v2_prefix(7), hlen);                                                                                                                                     
      |                                          ^~~~~~~~~~~~~~~~~~                                                                                                                                                
      |                                          ngx_http_v2_write_int                                                                                                                                             
gbuild/libnginx/origin/nginx-1.15.6/src/http/v2/ngx_http_v2_encode.c:31:12: error: ‘NGX_HTTP_V2_ENCODE_RAW’ undeclared (first use in this function; did you mean ‘NGX_HTTP_V2_BUFFERED’?                                                                                                                                                                             
   31 |     *dst = NGX_HTTP_V2_ENCODE_RAW;                                                                                                                                                                         
      |            ^~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                          
      |            NGX_HTTP_V2_BUFFERED
```

Config options:
* `LIBMUSL`
* `LIBNGINX`
* `LIBLWIP`
* `LIBNGINX_HTTP`

Note: Musl does not work with `uksignal` at the moment, so you will need to disable that.

**Comment by razvand** [comment]
Slved by #12

**Comment by razvand** [comment]
Solved by #12
